### PR TITLE
v6.0: Admin upgrade notice

### PR DESCRIPTION
**Pull request by esethna**
Rebasing Katie's PR: https://github.com/mattermost/notices/pull/219

Replacing the v5.39 notice with v6.0

### CODE REVIEW

**Comment by esethna** [comment]
FYI @amyblais this is the v6.0 upgrade notice. I will likely also be submitting some for end user features after I have the assets from design

File: notices.json [hunk 1]
```diff
@@ -37,21 +37,21 @@
...
-    "id": "server_upgrade_v5.39",
+    "id": "server_upgrade_v6.00",
     "conditions": {
       "audience": "sysadmin",
       "clientType": "all",
-      "serverVersion": ["<5.39"],
+      "serverVersion": ["<6.00"],
       "instanceType": "onprem",
-      "displayDate": ">= 2021-09-21T00:00:00Z"
+      "displayDate": ">= 2021-10-15T00:00:00Z"
```
Comment (amyblais) [inline notices.json:46]: 0/5, maybe Monday 18th is better than a Monday.

File: notices.json [hunk 2]
```diff
@@ -37,21 +37,21 @@
...
       "en": {
-        "title": "New server version available",
-        "description": "This new release includes bug fixes for increased stability and some important notices for upcoming product changes and events. [Upgrading](https://docs.mattermost.com/upgrade/upgrading-mattermost-server.html) only takes a few minutes.",
-        "image": "https://raw.githubusercontent.com/mattermost/notices/release/images/server_upgrade.png",
+        "title": "Mattermost 6.0 is here!",
+        "description": "Mattermost has expanded beyond channel-based messaging and offers Playbooks and Boards as core features to our open source developer collaboration platform!",
+        "image": "https://raw.githubusercontent.com/mattermost/notices/master/images/6.0.png",
         "actionText": "Learn more",
-        "actionParam": "https://mattermost.com/blog/mattermost-v5-39/"
+        "actionParam": "https://mattermost.com/blog/mattermost-v6-00/"
```
Comment (amyblais) [inline notices.json:54]: Will check URL once blog post is uploaded to Wordpress

File: notices.json (hunk 1 above)
Comment (esethna) [reply notices.json:46]: +1

**Comment by esethna** [comment]
@amyblais @ogi-m this is now ready for QA review, please help test and ensure it gets merged for the launch.

File: notices.json (hunk 2 above)
Comment (amyblais) [reply notices.json]: Fixed the link

**Pull request merged**